### PR TITLE
Loop balance and usd prices

### PR DESCRIPTION
**Pull request by ntropian**

### CODE REVIEW

File: obi-wallet/apps/mobile/ios/Mobile.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -355,7 +355,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "set -e\n\nexport NODE_BINARY=node\nexport ENTRY_FILE=${PROJECT_DIR}/../src/main.tsx\n../node_modules/react-native/scripts/react-native-xcode.sh\n";
+			shellScript = "set -e\n\nexport NODE_BINARY=/Users/ntropian/.nvm/versions/node/v16.13.2/bin/node\nexport ENTRY_FILE=${PROJECT_DIR}/../src/main.tsx\n../node_modules/react-native/scripts/react-native-xcode.sh\n";
```
Comment (inyono) [inline obi-wallet/apps/mobile/ios/Mobile.xcodeproj/project.pbxproj:358]: That shouldn't be part of the repo. I think there was a way to override Xcode settings w/o committing (Xcode env or something) but I can't remember the details :/

File: obi-wallet/apps/mobile/ios/Mobile.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -552,6 +553,7 @@
 			buildSettings = {
 				BUNDLE_LOADER = "$(TEST_HOST)";
 				COPY_PHASE_STRIP = NO;
+				DEVELOPMENT_TEAM = B364RPTB8B;
```
Comment (inyono) [inline obi-wallet/apps/mobile/ios/Mobile.xcodeproj/project.pbxproj:556]: Same. But does this fix the sign problem permanently?

**Review by inyono — CHANGES_REQUESTED** [review]

**Comment by inyono** [comment]
You can ignore the build error, that's on me.

File: obi-wallet/apps/mobile/ios/Mobile.xcodeproj/project.pbxproj (hunk 1 above)
Comment (ntropian) [reply obi-wallet/apps/mobile/ios/Mobile.xcodeproj/project.pbxproj:358]: Yeah, this was unintentional commit for sure, thanks.

**Pull request closed without merging**